Multisig/UI/UI Library/View Controllers/LoadableViewController/LoadableViewController.swift

```diff
@@ -25,6 +25,7 @@ class LoadableViewController: UIViewController {
     @IBOutlet weak var dataErrorView: ScrollableDataErrorView!
     @IBOutlet weak var emptyView: ScrollableEmptyView!
     @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var ribbonView: RibbonView!
```

Comment:
Why did you decide to use here a separate view instead of a wrapper view controller?

Reply:
It's a good point! I've tried with the wrapper VC now, and it works OK, so I'll use that.

I did it because I first implemented ribbonView like that in previous pull request.